programs/ssl/ssl_client2.c

```diff
@@ -549,6 +594,14 @@ struct options
                                  * after renegotiation                      */
     int reproducible;           /* make communication reproducible          */
     int skip_close_notify;      /* skip sending the close_notify alert      */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
+    const char *named_groups_string;    /* list of named groups             */
+    const char *key_share_named_groups_string;    /* list of named groups   */
+    int key_exchange_modes;     /* supported key exchange modes  */
+#if defined(MBEDTLS_ZERO_RTT)
+    int early_data;             /* support for early data */
```

Comment:
Minor: Keep the alignment:
```suggestion
    const char *named_groups_string;    /* list of named groups             */
    const char *key_share_named_groups_string;    /* list of named groups   */
    int key_exchange_modes;     /* supported key exchange modes             */
#if defined(MBEDTLS_ZERO_RTT)
    int early_data;             /* support for early data                   */
```